src/server/scripts/Northrend/Ulduar/Ulduar/boss_thorim.cpp

```diff
@@ -1422,6 +1377,10 @@ class boss_thorim_runic_colossus : public CreatureScript
             if (me->GetInstanceScript())
                 if (GameObject* go = ObjectAccessor::GetGameObject(*me, me->GetInstanceScript()->GetGuidData(DATA_THORIM_FIRST_DOORS)))
                     go->SetGoState(GO_STATE_ACTIVE);
+
+            if (me->GetInstanceScript())
+                if (Creature* cr = ObjectAccessor::GetCreature(*me, me->GetInstanceScript()->GetGuidData(TYPE_THORIM)))
+                    cr->AI()->Talk(SAY_SPECIAL_2);
```

Comment:
remove the first if statement and put the rest in the if statement of L#1377

Reply:
Wow, I am in fact blind, thanks.